README.md

```diff
@@ -1 +1,45 @@
 This is an Eclipse plugin for building and deploying Google Cloud Platform applications.
+
+* Development
+
+** Import into Eclipse
+
+*** Requirements
+
+1. Eclipse Mars or later
+
+1. The [m2e plugin](http://www.eclipse.org/m2e/) installed to import the project into Eclipse
+
+1. JDK 7
+
+*** Steps to import
+
+1. Setup JDK 7 in Eclipse
+  1. Select `Window/Preferences` (on Mac `Eclipse/Preferences`)
+  1. Under `Java/Installed JREs` click `Add`
+  1. Select Standard VM and click `Next`
+  1. Select the folder that contains the JDK 7 installation by clicking
+     `Directory`
+  1. Set the name to `jdk7`
+  1. Click `Finish`
+  1. Set the checkbox selected for the JDK you just added to mark it as default
+  1. Click `Apply`, then `OK`
+1. Select `File/Import...` menu in Eclipse
```

Comment:
We need a step to load the target file. Either by opening the `.target` file in the _Target Platform Editor_ (more direct), or by using _Preferences &rarr; Plug-in Development &rarr; Target Platforms_ and selecting the _Eclipse Mars 4.5.2_ target.